tools/packaging/static-build/qemu/Dockerfile

```diff
@@ -56,5 +61,6 @@ RUN PREFIX="${PREFIX}" /root/configure-hypervisor.sh -s kata-qemu | xargs ./conf
 	--with-pkgversion=kata-static
 
 RUN make -j$(nproc)
+RUN make -j$(nproc) virtiofsd
```

Comment:
When dealing with the one using DAX, we end up doing a `chmod +x ... && mv virtiofsd ...`. Just to be more than sure, that whole dance is not necessary here, right? And why do we do the whole dance when dealing with DAX?

Reply:
We don't need it beucase qemu by default installs virtiofsd there. We are doing that dance for -DAX version to avoid conflicts with non-dax virtiofs (just to be sure not to override any existing non-DAX virtiofsd).